titanic.ipynb

```diff
@@ -5,14 +5,16 @@
       "cell_type": "markdown",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 2 From line 18 to 19_
```python
# Show the plot
plt.show()
```
 my new review comment

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/25)._